docs/user/reference/resource-compute/list-of-commands-available-on-a-compute-resource-provisioned-by-juju/list-of-juju-introspect-macros/juju_agent.md

```diff
@@ -1,4 +1,55 @@
 (juju_agent)=
 # `juju_agent`
 
-TBA
+The juju_agent script is a proxy function which 
+will pass the arguments to the correct agent (
+machine, controller, application or unit).
```

Comment:
I'm afraid I still don't understand. What arguments? And how do we tell it which agent to pass it to?